src/index.html

```diff
@@ -10,23 +10,26 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search Bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form class="search-bar">
+      <input
+        type="text"
+        data-qa="big"
+        placeholder="Try “Los Angeles“"
+        class="search-bar__big input"
+      />
```

Comment:
According to the HTML Formatting checklist, it's recommended to add empty lines between multiline sibling blocks of HTML. This will make your code more readable. Please add an empty line between these two input fields.